# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cinelerra-gg/hotfix-install.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cinelerra-5.1/Makefile.am.vanilla	2026-02-05 04:10:20.443001871 +0000
+++ cinelerra-5.1/Makefile.am	2026-02-05 04:12:54.943006414 +0000
@@ -77,9 +77,9 @@
 	rm -f Makefile
 
 # with-libzmpeg3
-zmp_install:
+zmp_install: bin_install
 	cd bin; $(inst_sh) "$(DESTDIR)$(bindir)" \
-		zmpeg3{cc2txt,ifochk}
+		zmpeg3cc2txt  zmpeg3ifochk
 	cd bin; $(inst_sh) "$(DESTDIR)$(WANT_CINLIB_DIR)" \
 		hveg2enc mpeg2enc mplex mplexlo
 
@@ -87,9 +87,9 @@
 	rm -f "$(DESTDIR)$(bindir)"/zmpeg3{cc2txt,ifochk}
 
 # with-libzmpeg3 without thirdparty
-zmp_install_nothird:
+zmp_install_nothird: bin_install
 	cd bin; $(inst_sh) "$(DESTDIR)$(bindir)" \
-		zmpeg3{cc2txt,ifochk}
+		zmpeg3cc2txt  zmpeg3ifochk
 	cd bin; $(inst_sh) "$(DESTDIR)$(WANT_CINLIB_DIR)" \
 		hveg2enc mplexlo
 
@@ -98,7 +98,7 @@
 
 
 # with-commercial
-com_install:
+com_install: bin_install
 	cd bin; $(inst_sh) "$(DESTDIR)$(bindir)" cin_db
 	cd bin; $(inst_sh) "$(DESTDIR)$(WANT_CINLIB_DIR)" cutads
 
@@ -106,14 +106,14 @@
 	rm -f "$(DESTDIR)$(bindir)/cin_db"
 
 # with-ladspa
-lad_install:
+lad_install: bin_install
 	cd bin/ladspa; $(inst_sh) "$(DESTDIR)$(WANT_LADSPA_DIR)" .
 
 lad_uninstall:
 	rm -rf "$(DESTDIR)$(WANT_LADSPA_DIR)"
 
 # with-lv2
-lv2_install:
+lv2_install: bin_install
 	cd bin; $(inst_sh) "$(DESTDIR)$(WANT_CINLIB_DIR)" lv2ui
 	cd bin; $(inst_sh) "$(DESTDIR)$(datadir)/$(WANT_CIN)" \
 		lv2 lv2_blacklist.txt
